pkg/pgmodel/ingestor/metric_batcher.go

```diff
@@ -32,55 +32,62 @@ type readRequest struct {
 	copySender <-chan copyRequest
 }
 
-func metricTableName(conn pgxconn.PgxConn, metric string) (string, bool, error) {
+func metricTableName(conn pgxconn.PgxConn, metric string) (int64, string, bool, error) {
 	res, err := conn.Query(
 		context.Background(),
 		getCreateMetricsTableWithNewSQL,
 		metric,
 	)
 
 	if err != nil {
-		return "", true, fmt.Errorf("failed to get the table name for metric %s: %w", metric, err)
+		return 0, "", true, fmt.Errorf("failed to get the table name for metric %s: %w", metric, err)
 	}
 
-	var tableName string
-	var possiblyNew bool
+	var (
+		metricID    int64
+		tableName   string
+		possiblyNew bool
```

Comment:
Can we use named returns so it helps in making this func be cleaner? It will avoid making `0, ` in the entire func. Just a simple `return`

Reply:
No, naked returns are a source of confusion and make the code harder to read especially when its not a trivial function. They should be avoided in most cases.

Can switch this to named returns so we don't need to declare new vars.